mockOSApi.v2/Controllers/ProcessController.cs

```diff
@@ -90,18 +139,23 @@ public ActionResult<MockProcessDto> Get(int pid)
     /// <returns></returns>
     /// 
     [HttpPost]
-  //  [Authorize("admin_policy")] // to implement authorization with JWT separately
     public async Task<ActionResult<MockProcessDto>> CreateProcess(MockProcessCreationDto process)
     {
+        var token = FetchTokenFromHeaders();
+        if (!_authorizationService.IsUserAuthorized(token))
+            return Unauthorized();
+        var user = _authorizationService.GetUser(token);
+
 #pragma warning disable CS8602 // Dereference of a possibly null reference.
```

Comment:
I don't think you need this anymore. You did a null check right below for process.
If this is because user is null, then you should do a null check on user